specification/quantum/resource-manager/Microsoft.Quantum/preview/2019-11-04-preview/examples/providerList.json

```diff
@@ -0,0 +1,87 @@
+{
+  "parameters": {
+    "subscriptionId": "1C4B2828-7D49-494F-933D-061373BE28C2",
+    "api-version": "2019-11-04-preview",
+    "locationName": "westus2"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "QIO",
```

Comment:
this should be the full resource id /subscriptions/{subId}/providers/Microsoft.Quantum/{offeringsOrProviders}/{name}

Reply:
these are not ARM resources, as such they don't have a full resourceId.

Reply:
they must be ARM resources since they are exposed via an ARM api

Reply:
+1

---
In reply to: [546981115](https://github.com/Azure/azure-rest-api-specs/pull/11266#discussion_r546981115) [](ancestors = 546981115)

Reply:
Just hopped off a call with the RP team, this is a metadata kind of API like the skus api or marketplace image API. this is not something which will be created by the end user. They can only do a GET on this. So, it need not comply with the ARM resource schema if it doesnt make sense. The RP team will change the schema so that its not confused with an ARM resoruce schema (remove the id, name etc properties and call it something else). They will make this chanhge after prview.